pwiz_tools/Skyline/Controls/Graphs/AllChromatogramsGraph.cs

```diff
@@ -470,14 +470,11 @@ private void AddProgressControls(MultiProgressStatus status)
                 progressControl.ShowGraph += (sender, args) => ShowGraph();
                 progressControl.ShowLog += (sender, args) => ShowLog();
                 controlsToAdd.Add(progressControl);
+                _fileProgressControls.Add(filePath.GetLocation(), progressControl);

```

Comment:
This seems to be a repeat or removal of Nick's recent change and not related to this PR.